programs/magicblock/src/schedule_task/process_schedule_task.rs

```diff
@@ -61,11 +60,12 @@ pub(crate) fn process_schedule_task(
     }
 
     // Enforce minimal execution interval
-    if args.execution_interval_millis < MIN_EXECUTION_INTERVAL {
+    let min_interval = min_task_scheduler_interval();
+    if args.execution_interval_millis < min_interval {
         ic_msg!(
             invoke_context,
             "ScheduleTask ERR: execution interval must be at least {} milliseconds",
-            MIN_EXECUTION_INTERVAL
+            min_interval
         );
         return Err(InstructionError::InvalidInstructionData);
```

Comment:
And in general we should consider silent clamping of the value, documenting that behavior in examples for scheduled tasks.